redwood-tooling-codegen/src/main/kotlin/app/cash/redwood/tooling/codegen/sharedHelpers.kt

```diff
@@ -101,7 +101,7 @@ internal fun Schema.getWidgetFactoryType(): ClassName {
 }
 
 internal fun Schema.getMutableWidgetFactoryType(): ClassName {
-  return ClassName(widgetPackage(), "Mutable${type.flatName}WidgetFactory")
+  return ClassName(widgetPackage(), "${type.flatName}MutableWidgetFactory")
```

Comment:
This reads gooder. "Mutable" is operating on "Widget", not the whole factory. Need to make the same change for the protocol factory separately.